src/kili/graphql_client.py

```diff
@@ -90,9 +90,9 @@ def _send(self, query, variables):
             req = None
             try:
                 try:
-                    number_of_trials = int(os.getenv("KILI_SDK_TRIALS_NUMBER", "10"))
+                    number_of_trials = int(os.getenv("KILI_SDK_TRIALS_NUMBER", "1"))
```

Comment:
this was for debug purposes ?

Reply:
It was for the POC, we don't need it for the beta. I'll remove that